deploy/gallery/create_service.sh

```diff
@@ -0,0 +1,61 @@
+#! /usr/bin/env bash
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+# This script creates the required linked service & the spark 
+# job definition on the Synapse workspace and should be run 
+# prior to executing the pipeline template from Synapse gallery.
+
+set -e
+
+ENV_CODE=${1:-${ENV_CODE}}
+
+if [[ -z ${ENV_CODE} ]]
+  then
+    echo "Missing Environment Code"
+    exit 1
+fi
+
+base_dir="$(cd "$(dirname "$0")/../../" && pwd)"
```

Comment:
can we call it project_root_dir, preferrably in caps ... PROJECT_ROOT_DIR, base_dir does not represents what we want to convey.

In other scripts we have used `PRJ_ROOT="$(cd `dirname "${BASH_SOURCE}"`/..; pwd)"`

Reply:
Will change the variable name later. We want to merge this change to support the pipeline template. `base_dir` is the root directory of the project. As it is a local variable, I'd prefer to keep it lower case because the upper case is only used for environment variables in bash.